tests/fuzz/22.c

```diff
@@ -139,7 +139,7 @@ static uint64_t  func_60(uint32_t  p_61, uint32_t  p_62, uint32_t * p_63, const
 
```

Comment:
Maybe revert tests/fuzz/...? this looks like auto-generated code so I don't think it makes sense to edit like this (or hold it to the same standards).